admin/model_api_atlas_fts_analyzers_char_filters_inner_mappings.go

```diff
@@ -0,0 +1,115 @@
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package admin
+
+import (
+	"encoding/json"
+)
+
+// checks if the ApiAtlasFTSAnalyzersCharFiltersInnerMappings type satisfies the MappedNullable interface at compile time
+var _ MappedNullable = &ApiAtlasFTSAnalyzersCharFiltersInnerMappings{}
```

Comment:
I found this unusable and since it contains only additional properties the whole objects doesn't make sense. 
WIth those views being manual we are very likely to get breaking changes 100%